Assets/Scripts/PickupAmmunition.cs

```diff
@@ -0,0 +1,34 @@
+using UnityEngine;
+
+public class PickupAmmunition : MonoBehaviour
+{
+    public Transform hand;
+    public Vector3 inHandSize = new Vector3(0.2f, 0.2f, 0.2f);
+    public GameObject projectileHitParticles;
+
+    public Ammunition ammunition;
+
+    void Update()
+    {
+        if (!ammunition
+            && Input.GetButtonDown("Fire2")
+            && Physics.Raycast(Camera.main.ScreenPointToRay(Input.mousePosition), out var hitInfo, 2f, LayerMask.NameToLayer("Player")))
```

Comment:
Suggest extracting some of these values as public fields. Namely the distance and layer mask.